### ISSUE TITLE
Vulcanize breaks templates within tables

### ISSUE THREAD

**Issue by rezonant** [Polymer/vulcanize#87]
I have created a repository with a basic Polymer/vulcanize project to demonstrate this issue.
https://github.com/rezonant/vulcanize-tables-bugs

It appears that repeat templates within tables do not expand correctly after vulcanization.
This appears to be because vulcanize's DOM parsing applies the same rules to elements that browsers do. While it works when it happens at run time, doing it prior to run time loses meaning. For instance, where the original code was:

        <table>
            <thead>
                <tr>
                    <th>ID</th>
                    <th>Name</th>
                    <th>Action</th>
                </tr>
            </thead>
            <tbody>
                <template repeat="{{data.results}}">
                    <tr>
                        <td>{{id}}</td>
                        <td>{{name}}</td>
                        <td>{{action}}</td>
                    </tr>
                </template>
            </tbody>
        </table>

Vulcanize would then produce:

        <template repeat="{{data.results}}">
                    </template><table>
            <thead>
                <tr>
                    <th>ID</th>
                    <th>Name</th>
                    <th>Action</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                        <td>{{id}}</td>
                        <td>{{name}}</td>
                        <td>{{action}}</td>
                    </tr>

            </tbody>
        </table>

When Polymer interprets this later, the original meaning is lost.

**Comment by azakus** [comment]
This should be fixed in 0.7. However, using `<td template>` and `<option template>` is still required for any browsers that do not support the `<template>` tag natively.

**Comment by hungyao** [comment]
What's the solution for Polymer >= 1?  It fails in exactly the same way as the original poster.

e.g.:
```
      <table>
        <thead>
            <tr>
                <th>ID</th>
                <th>Name</th>
                <th>Action</th>
            </tr>
        </thead>
        <tbody>
            <template is="dom-repeat" items="{{data.results}}" as="result">
                <tr>
                    <td>{{result.id}}</td>
                    <td>{{result.name}}</td>
                    <td>{{result.action}}</td>
                </tr>
            </template>
        </tbody>
    </table>
```
